newdle/client/src/components/TopHeader.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {Grid, Header} from 'semantic-ui-react';
+
+import styles from '../App.module.scss';
+
+import Navigator from './Navigator.js';
+
+const steps = [
```

Comment:
How can I only render the active step's body card that way without having everything appear in the wrong place?
The three step numbers on top should always be rendered next to each other and then only one component with details should appear underneath them.

Reply:
If you use the first version then you simply don't render `children` unless the `active` prop is set.

Reply:
I have to render the numbers no matter what, though. Should those number boxes be outside of Navigator?

Reply:
ah damn, didn't think of those... maybe there's some react magic that can be used  to know the index...  anyway, the easiest option for now is just to have `n={1}`, `n={2}` and `n={3}` on the `<Navigator.Step />` components

Reply:
actually CSS can display the numbers, no need to know them in the JS code at all: https://codepen.io/anon/pen/VojNrj

Reply:
Shouldn't the numbered boxes be rendered first and then whichever one is active also has its title/description part rendered? That has the logic of the numbered box and the title/description separately, kind of like how Tabs work.

Reply:
you're right.. I just checked how SUI does it for their tabs and turns out they also use a mapping array instead of something nicer with react components..

Reply:
Maybe let's stick with the array then and just get rid of the counter in there (since that is really not needed as the array index + 1 is already a good counter)...